### PR TITLE
Added a alert

### PR DESCRIPTION
**Pull request by eevargas**
Added a alert when clicked on the title

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
 (function () {
-
+    document.getElementById("page-title")
+        .addEventListener("click", () => {
+            alerts("Hello")
```
Comment (ralmidani) [inline assets/js/main.js:4]: Did you mean `alert`? (Don't worry, I've made this mistake a few times)

**Review by ralmidani — CHANGES_REQUESTED** [review]
Please see in-line comment

File: assets/js/main.js (hunk 1 above)
Comment (eevargas) [reply assets/js/main.js]: ok... thanks i missed that.

**Review by ralmidani — APPROVED** [review]
Thanks for fixing!

**Pull request merged**